tests/golden/check-local-refs/dir1/dir2/d2f1.md

```diff
@@ -0,0 +1,35 @@
+<!--
+ - SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -->
+
+# Current file links
+[existing-cf-ref](#current-file-links)
+[bad-cf-ref](#bad)
+
+# Relative links
+[existing-file-rel](d2f2.md)
+[existing-file-rel](./d2f2.md)
+[existing-file-rel](../dir2/.././d1f1.md)
+
+
+[existing-anchor-rel-1](d2f2.md#existing-anchor-d2f2)
+[existing-anchor-rel-2](./d2f2.md#existing-anchor-d2f2)
+[existing-anchor-rel-3](../dir2/../d1f1.md#existing-anchor-d1f1)
+
+[bad-file-rel](../a/b/c/unexisting-file.md)
+
+[bad-anchor-rel-1](d2f2.md#bad-anchor)
+[bad-anchor-rel-2](unexisting-file.md#bad-anchor)
+
+# Absolute links
+Should be correct when root is `/tests/golden/check-local-refs`:
+[file-abs-1](/dir1/./d1f1.md)
+[anchor-abs-1](/dir1/.././d1f1.md#existing-anchor-d1f1)
```

Comment:
This link isn't valid (and it does indeed appear as an invalid link in `expected1.gold`), but in line 27 we say that it "Should be correct".

Reply:
My bad, thanks, fixed